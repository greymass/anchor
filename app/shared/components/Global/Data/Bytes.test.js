import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalDataBytes from './Bytes';

const mockProps = {
  bytes: 1000,
};

describe('GlobalDataBytes', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalDataBytes {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
