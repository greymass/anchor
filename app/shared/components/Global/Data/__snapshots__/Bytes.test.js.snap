// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalDataBytes renders correctly 1`] = `
<Popup
  content="1000 B"
  inverted={true}
  keepInViewPort={true}
  on="hover"
  position="top left"
  trigger={
    <span>
       1 kB 
    </span>
  }
/>
`;
